README.md

```diff
@@ -5,4 +5,24 @@
 
 # snp
 
+The `snp` crate provides an implementation of [AMD Secure Encrypted
+Virtualization - Secure Nested Paging (SEV-SNP)] APIs.
```

Comment:
Is this meant to be a link to an SNP page?

Reply:
Yea, i'll add in a link